vwifi.c

```diff
@@ -403,24 +430,169 @@ static struct cfg80211_ops owl_cfg_ops = {
     .get_station = owl_get_station,
 };
 
+int owl_ndo_open(struct net_device *dev)
+{
+    /* The first byte is '\0' to avoid being a multicast
+     * address (the first byte of multicast addrs is odd).
+     */
+    memcpy(dev->dev_addr, "\0owl0", ETH_ALEN);
```

Comment:
How can you ensure that interface is always `owl0`? If we allow multiple `owl` interfaces, `owl1` or `owl2` might appear.